src/core/models.py

```diff
@@ -112,25 +136,50 @@ def chroot_acl(self) -> str:
             - 'w' - write
             - 'a' - append
         """
-        return self.relation_data.get("chroot-acl", "cdrwa")
+        return self.relation_data.get(
+            "chroot-acl", "cdrwa"
+        )  # pyright: ignore reportGeneralTypeIssues
 
     @property
     def chroot(self) -> str:
         """The client requested root zNode path value."""
         chroot = self.relation_data.get("chroot", "")
-        if not chroot.startswith("/") and chroot:
+        if chroot and not chroot.startswith("/") and chroot:
```

Comment:
**issue**: there is  a duplicate `chroot` in the condition

Reply:
Yeah, we can change it to `if chroot and not chroot.startswith("/")`